File: 01_gpio_digital/ledblink.c
#include <wiringPi.h>
#define LED_PIN 4

int main(void)
{
    //wiringPiSetup();
    wiringPisetupGpio();
    pinMode(LED_PIN, OUTPUT);
    for (int i=0; i<5; i++)
    {
        digitalWrite(LED_PIN, HIGH); delay(5000);
        digitalWrite(LED_PIN, LOW); delay(5000);
    }
    return 0;
}

File: 01_gpio_digital/ledblink.py
import RPi.GPIO as GPIO
import time

LED_PIN = 4
GPIO.setmode(GPIO.BCM)
GPIO.setup(LED_PIN, GPIO.OUT)

for i in range(10):
    GPIO.output(LED_PIN, GPIO.HIGH)
    print("led on")
    time.sleep(1)
    GPIO.output(LED_PIN, GPIO.LOW)
    print("led off")
    time.sleep(1)

GPIO.cleanup()
